core/src/commonMain/kotlin/com/xebia/functional/xef/llm/Embeddings.kt

```diff
@@ -1,30 +1,29 @@
 package com.xebia.functional.xef.llm
 
 import arrow.fx.coroutines.parMap
-import com.xebia.functional.xef.llm.models.embeddings.Embedding
-import com.xebia.functional.xef.llm.models.embeddings.EmbeddingRequest
-import com.xebia.functional.xef.llm.models.embeddings.EmbeddingResult
-import com.xebia.functional.xef.llm.models.embeddings.RequestConfig
+import com.xebia.functional.openai.apis.EmbeddingsApi
+import com.xebia.functional.openai.models.CreateEmbeddingRequest
+import com.xebia.functional.openai.models.CreateEmbeddingRequestModel
+import com.xebia.functional.openai.models.Embedding
+import com.xebia.functional.openai.models.ext.embedding.create.CreateEmbeddingRequestInput
 
-interface Embeddings : LLM {
-  suspend fun createEmbeddings(request: EmbeddingRequest): EmbeddingResult
+suspend fun EmbeddingsApi.embedDocuments(texts: List<String>, chunkSize: Int?): List<Embedding> =
+  if (texts.isEmpty()) emptyList()
+  else
+    texts
+      .chunked(chunkSize ?: 400)
```

Comment:
Maybe can be useful move this 400 as a default value of the input parameter chunkSize.